recipes/03/03_group_by.md

```diff
@@ -0,0 +1,43 @@
+# 03 Aggregating Data
+
+:bulb: This example will show how to aggregate server logs in real-time using the standard `GROUP BY` clause.
+
+The source table (`server_logs`) is backed by the [`faker` connector](https://github.com/knaufk/flink-faker), which continuously generates rows in memory based on Java Faker expressions.
+
+To count the number of logs received for each status code _over time_, you can combine the `COUNT` aggregate function with a `GROUP BY` clause. A `GROUP BY` on a streaming table produces an updating result, so you will see the aggregated count for each status code continuously changing as new rows flow in.
+
+> As an exercise, you can play around with other standard SQL aggregate functions (e.g. `SUM`,`AVG`,`MIN`,`MAX`).
+
+## Script
+
+```sql
+CREATE TABLE server_logs ( 
+    client_ip STRING,
+    client_identity STRING, 
+    userid STRING, 
+    log_time TIMESTAMP(3),
+    request_line STRING, 
+    status_code STRING, 
+    size INT
+) WITH (
+  'connector' = 'faker', 
+  'fields.client_ip.expression' = '#{Internet.publicIpV4Address}',
+  'fields.client_identity.expression' =  '-',
+  'fields.userid.expression' =  '-',
+  'fields.log_time.expression' =  '#{date.past ''15'',''5'',''SECONDS''}',
+  'fields.request_line.expression' = '#{regexify ''(GET|POST|PUT|PATCH){1}''} #{regexify ''(/search\.html|/login\.html|/prod\.html|cart\.html|/order\.html){1}''} #{regexify ''(HTTP/1\.1|HTTP/2|/HTTP/1\.0){1}''}',
+  'fields.status_code.expression' = '#{regexify ''(200|201|204|400|401|403|301){1}''}',
+  'fields.size.expression' = '#{number.numberBetween ''100'',''10000000''}'
+);
+
+SELECT status_code, 
```

Comment:
nit: align indentation with the other examples like https://github.com/ververica/flink-sql-cookbook/blob/master/recipes/02/02_where.md.